File: web-admin/src/main/java/com/isystk/sample/web/admin/controller/html/system/mail/SystemMailSearchForm.java
package com.isystk.sample.web.admin.controller.html.system.mail;

import com.isystk.sample.common.dto.Pageable;
import com.isystk.sample.web.base.controller.html.BaseSearchForm;
import javax.validation.constraints.Digits;

public class SystemMailSearchForm extends BaseSearchForm implements Pageable {

  private static final long serialVersionUID = 7593564324192730932L;

  @Digits(integer = 9, fraction = 0)
  Integer templateDiv;

  public @Digits(integer = 9, fraction = 0) Integer getTemplateDiv() {
    return this.templateDiv;
  }

  public void setTemplateDiv(@Digits(integer = 9, fraction = 0) Integer templateDiv) {
    this.templateDiv = templateDiv;
  }
}

File: web-admin/src/main/kotlin/com/isystk/sample/web/admin/controller/html/system/mail/SystemMailSearchForm.kt
package com.isystk.sample.web.admin.controller.html.system.mail

import com.isystk.sample.common.dto.Pageable
import com.isystk.sample.web.base.controller.html.BaseSearchForm
import javax.validation.constraints.Digits

class SystemMailSearchForm : BaseSearchForm(), Pageable {
    var templateDiv: @Digits(integer = 9, fraction = 0) Int? = null

    companion object {
        private const val serialVersionUID = 7593564324192730932L
    }
}